### ISSUE TITLE
[HOLD #22142] [$1000] Web - Triple clicking on an edited text selects the "edited" label

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#23402]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Open a chat
2. Post a comment
2. Edit the last comment you posted
3. Triple click on the comment to select it

## Expected Result:
The message is selected

## Actual Result:
The edited label is selectable

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [x] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.43-6
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/6d5f4c7e-2bfd-4736-8f55-7d0a9aec812a

**Expensify/Expensify Issue URL:**
**Issue reported by:** @daveSeife
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1689887962165499

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01bee694b754e4a650</li>
        <li>Upwork Job ID: 1683654217343696896</li>
        <li>Last Price Increase: 2023-07-25</li>
    </ul>
</details>

**Comment by neonbhai** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
Triple clicking on an edited text selects the "edited" label

### What is the root cause of that problem?
The `(edited)` message is rendering via this code. The enclosing text tag is missing the `selectable={false}` prop and `userSelectNone` style.

### What changes do you think we should make in order to solve the problem?
We should set the `selectable` prop as `false`, and add the `userSelectNone` style to the enclosing `<Text>` tag.

### What alternative solutions did you explore? (Optional)
xx

**Comment by sophiepintoraetz** [comment]
Able to reproduce this
![2023-07-25_13-41-56 (1)](https://github.com/Expensify/App/assets/42940078/28e4c107-3026-40d2-a58b-3fa22d90ac14)

**Comment by sophiepintoraetz** [comment]
Not overdue - waiting on proposals

**Comment by jigartankhupp** [comment]
### **Proposal**
- - -

### **Please re-state the problem that we are trying to solve in this issue.**
Triple clicking on an edited text selects the "edited" label

- - -
### **What is the root cause of that problem?**
The root cause of the text selection behavior when double-clicking or triple on text in a browser is the default behavior of the browser itself. When we double-click or triple on text in most modern browsers, the default behavior is to select the word or paragraph that we have double-clicked or triple on. This behavior is implemented by the browser's built-in user interface and is designed to make it easy for users to select text for copying, pasting, or other operations.
- - -
### **What changes do you think we should make in order to solve the problem?**
To solve the problem and disable text selection when double-clicking or triple on text in our CSS code, we can make the following changes:
```
-moz-user-select:none; /* firefox */
-webkit-user-select: none; /* Safari */
-ms-user-select: none; /* IE 10 and IE 11 */
user-select: none; /* Standard syntax */
```
**Final Result**

https://github.com/Expensify/App/assets/38810839/ee1a5733-6e23-4421-97b6-361995951b79


- - -
### **What alternative solutions did you explore? (Optional)**
We can add the `selectable={false} & userSelect="none"` in the outer `<Text />` component. https://reactnative.dev/docs/text#selectable. Also we can remove the `selectable={false}` props from the inner `<Text />` Component.

I hope my proposal will accept.
Thanks

Contributor details
Your Expensify account email: [jigarjs@hupp.in](mailto:jigarjs@hupp.in)
Upwork Profile Link: https://www.upwork.com/freelancers/~012a5ed1b4a78d3b2c

**Comment by jigartankhupp** [comment]
Contributor details
Your Expensify account email: jigarjs@hupp.in
Upwork Profile Link: https://www.upwork.com/freelancers/~012a5ed1b4a78d3b2c

**Comment by sophiepintoraetz** [comment]
@0xmiroslav - mind reviewing the proposal, please?

**Comment by sophiepintoraetz** [comment]
Holding for https://github.com/Expensify/App/issues/22142

**Comment by sophiepintoraetz** [comment]
Holding for https://github.com/Expensify/App/issues/22142

**Comment by sophiepintoraetz** [comment]
Dropping to weekly while we're waiting on the hold.

**Comment by sophiepintoraetz** [comment]
Dropping to monthly - the referenced issue is on hold for another bug.

**Comment by sophiepintoraetz** [comment]
Still holding

**Comment by sophiepintoraetz** [comment]
Still holding.

**Comment by sophiepintoraetz** [comment]
holding still

**Comment by sophiepintoraetz** [comment]
I'm going to close this issue because 1) the issue we were held on is no longer reproducible and 2) this is does not affect a core flow.
